spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/config/SkipperServerPlatformConfiguration.java

```diff
@@ -46,36 +52,59 @@ public DeployerInitializationService deployerInitializationService(DeployerRepos
 		return new DeployerInitializationService(deployerRepository, platforms, resolver);
 	}
 
-	@Bean
 	@Profile("local")
```

Comment:
**[UNRELATED]** Is skipper only configured for local profile?

Reply:
By default, its local. The `kubernetes` or `cloud` profiles are set at runtime if deployed to other platforms.